actionview/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+*   Disable `ActionView::Template` finalizers in test environment
+
+    Template finalization can be expensive in large view test suites.
+    Add a configuration option,
+    `action_view.finalize_compiled_template_methods`, and turn it off in
```

Comment:
This reminder me that we need to update the configuration guide to add this new option. Can you do that?